VerilogCodingStyle.md

```diff
@@ -2291,6 +2292,109 @@ always_comb begin
 end
 ```
 
+### Functions and Tasks
+
+The following section applies to synthesizable RTL only.
+
+***In synthesizable RTL use of functions is allowed, provided they are declared
+automatic. Tasks should not be used.***
+
+Functions must be declared outside of a class in either a package or inside a
+module. A package is appropriate where the function relates to other definitions
```

Comment:
Wrap `package` & `module` in back-ticks since they are SV keywords?

Reply:
+1

Reply:
I decided against this as there's plenty of other places within the style guide that refer to module and package without the back-ticks. I think it partially comes down to keyword as noun vs keyword as keyword:

> Foos can only be declared within a package

vs

> There must only be one usage of `package` within a file

I will have a scan through this PR and identify any keyword as keyword usages and add back-ticks if they don't already exist.